opcua/uatypes.py

```diff
@@ -611,6 +616,9 @@ def _guess_type(self, val):
             return VariantType.ByteString
         elif type(val) == datetime:
             return VariantType.DateTime
+        elif type(val) == bool:
+            # TDA, added this because it was missing and causes exceptions when 'bool' type is used
```

Comment:
please do not write that kind of comments. only write comments when there is osmething which is difficutl to understand by reading the code